github/libp2p.yml

```diff
@@ -2119,6 +2119,11 @@ repositories:
         required_status_checks:
           strict: false
     collaborators:
+      admin:
+        - aschmahmann
+        - MarcoPolo
+        - Stebalien
```

Comment:
@Stebalien do you think it would still be helpful to continue to have admin here? At the moment I have you on since of the existing admin's that are not from Shipyard you're the only one whose been involved in the code at all.

If not can switch to maintain or leave you with push via Repos - Go.

Reply:
The only non-push thing I do is:

1. Cleanup branches (should be allowed in `Repos - Go`).
2. Delete spam and ban spammers.

Did we ever find a way to allow everyone to do the latter? If so, `Repos - Go` seems reasonable.

Reply:
> Delete spam

IIUC it looks like spam comments is doable with just write permissions to the repo https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments

> ban spammers

This can be done with an org-wide moderator role which you already have by virtue of being in the github-mgmt stewards group https://github.com/libp2p/github-mgmt/blob/989aed4f3864e6a264e62df4463b40978b7c01b4/github/libp2p.yml#L5077

As a result, I will downgrade to Repos Go